docs/source/contribs/development_MacM1.rst

```diff
@@ -0,0 +1,103 @@
+Run TensorCircuit on TensorlowBackend with Apple M1 Pro
```

Comment:
delete Pro throughout the note, as the problem are universal for M1 (pro/max/ultra)